modules/logging-release-notes-5-9-7.adoc

```diff
@@ -0,0 +1,30 @@
+// module included in logging-5-9-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-9-7_{context}"]
+= Logging 5.9.7
+This release includes link:https://access.redhat.com/errata/RHBA-2024:7324[OpenShift Logging Bug Fix Release 5.9.7].
+
+
+[id="openshift-logging-5-9-7-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, the `clusterlogforwarder.spec.outputs.http.timeout` parameter was not applied to the Fluentd configuration when Fluentd was used as the collector type, causing HTTP timeouts to be misconfigured. With this update, the `clusterlogforwarder.spec.outputs.http.timeout` parameter is now correctly applied, ensuring Fluentd honors the specified timeout and handles HTTP connections according to the user’s configuration. (link:https://issues.redhat.com/browse/LOG-6125[LOG-6125])
```

Comment:
- It would be better to avoid the passive voice in "was used as the collector type" if you can point to what uses Fluentd as the collector type
- "The user's configuration" is ambiguous - you mean configuration in the user account or you mean configuration that the user has provided?

Reply:
Thank you for your feedback. It totally makes sense. However, considering the current situation I have to move this PR for merge review. Having said that, I will be creating a Jira for this.

Reply:
Created Jira: [OBSDOCS-1364](https://issues.redhat.com/browse/OBSDOCS-1364)